chapters/robots.adoc

```diff
@@ -53,4 +53,4 @@ The type of wireless communication may also be restricted by the <<Local Organiz
 NOTE: Bluetooth is not allowed since it cannot be fixed to frequency channels.
 
 ==== Autonomy
-The robotic equipment has to be fully autonomous. Human operators are not permitted to enter any information to the system during a match, except in <<Overview, breaks>> or during a <<Timeouts,timeout>>. Disregarding this rule is considered <<Unsporting Behavior, unsporting behavior>>.
\ No newline at end of file
+The robotic equipment has to be fully autonomous. Human operators are not permitted to enter any information to the system during a match, except in <<Overview, breaks>> or during a <<Timeouts,timeout>>. Additionally, robots may not be manually moved during any phase of the match, except for moving robots out of the field, or into the field as described in <<Robot Substitution, Robot Substitution>>. Disregarding this rule is considered <<Unsporting Behavior, unsporting behavior>>.
```

Comment:
Small tip: It is good practice to put each sentence on a separate line. That's easier to read and makes it easier to compare the GIT diff.